user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/registration/MeasurementUpload.java

```diff
@@ -0,0 +1,424 @@
+
+      var uploadedItemsSectionTitle = new Span("Uploaded files");
+      uploadedItemsSectionTitle.addClassName("section-title");
+
+      uploadedItemsDisplays = new Div();
+      uploadedItemsDisplays.addClassName("uploaded-measurement-items");
+      uploadedItemsSection.add(uploadedItemsSectionTitle, uploadedItemsDisplays);
+      add(uploadSection, uploadedItemsSection);
+      addClassName("upload-items-display");
+    }
+
+    private void addFileToDisplay(MeasurementFileDisplay measurementFileDisplay) {
+      uploadedItemsDisplays.add(measurementFileDisplay);
+    }
+
+    private void removeFileFromDisplay(String fileName) {
+      MeasurementFileDisplay[] fileDisplays = fileDisplaysWithFileName(fileName);
+      uploadedItemsDisplays.remove(fileDisplays);
+    }
+
+    private MeasurementFileDisplay[] fileDisplaysWithFileName(String fileName) {
+      return uploadedItemsDisplays.getChildren()
+          .filter(MeasurementFileDisplay.class::isInstance)
+          .map(MeasurementFileDisplay.class::cast)
+          .filter(measurementFileDisplay -> measurementFileDisplay.measurementFileItem().fileName()
+              .equals(fileName))
+          .toArray(MeasurementFileDisplay[]::new);
+    }
+
+    public void hide() {
+      uploadedItemsDisplays.setVisible(false);
+    }
+
+    public void show() {
+      uploadedItemsDisplays.setVisible(true);
+    }
+  }
+
+  record MeasurementValidationReport(int validatedRows,
+                                     ValidationResult validationResult) {
+
+  }
+
+  public record MeasurementFileItem(String fileName,
+                                    MeasurementValidationReport measurementValidationReport) {
+
+  }
+
+  /**
+   * used to display an uploaded measurement file with validation information
+   */
+  public static class MeasurementFileDisplay extends Div {
+
+    public static final String SECONDARY = "secondary";
+
+    @Serial
+    private static final long serialVersionUID = -9075627206992036067L;
+    private final transient MeasurementFileItem measurementFileItem;
+    private final Div displayBox = new Div();
+
+    public MeasurementFileDisplay(MeasurementFileItem measurementFileItem) {
+      this.measurementFileItem = requireNonNull(measurementFileItem,
+          "measurementFileItem must not be null");
+      var fileIcon = VaadinIcon.FILE.create();
+      fileIcon.addClassName("file-icon");
+      Span fileNameLabel = new Span(fileIcon, new Span(this.measurementFileItem.fileName()));
+      fileNameLabel.addClassName("file-name");
+      add(fileNameLabel);
+      createDisplayBox(measurementFileItem.measurementValidationReport());
+      displayBox.addClassName("validation-display-box");
+      add(displayBox);
+      addClassName("measurement-item");
+    }
+
+    public MeasurementFileItem measurementFileItem() {
+      return measurementFileItem;
+    }
+
+    private void createDisplayBox(MeasurementValidationReport measurementValidationReport) {
```

Comment:
Is this creating a display box? 
```suggestion
    private void setDisplayBoxContent(MeasurementValidationReport measurementValidationReport) {
```
```suggestion
    private void fillDisplayBox(MeasurementValidationReport measurementValidationReport) {
```